src/components/Toolbar/Toolbar.js

```diff
@@ -247,10 +247,9 @@ export default class Toolbar extends Component {
 
         <Transition className={namespace} component="span">
           {isPanelActive && this.state.showContent ? (
-            <aside
-              className={`${namespace}__panel`}
+            <div
```

Comment:
@SimonFinney I noticed that you changed the `nav` element to a `div` -- while that removes the IBMa / WCAG warning, I question whether that's the best route to take here. Considering that this is a navigation list, it would be helpful to provide that info to assistive tech by using semantic markup.